iris/src/iris/domain/status/chat_status_update_dto.py

```
@@ -1,14 +1,14 @@
+from memiris.api.memory_dto import MemoryDTO
 from pydantic import Field
 
 from iris.domain.status.status_update_dto import StatusUpdateDTO
 
 
-class CourseChatStatusUpdateDTO(StatusUpdateDTO):
+class ChatStatusUpdateDTO(StatusUpdateDTO):
     result: Optional[str] = None
-    suggestions: List[str] = []
     session_title: Optional[str] = Field(alias="sessionTitle", default=None)
+    suggestions: List[str] = []
```

Ruff (0.15.9)

[warning] Mutable default value for class attribute

(RUF012)